packages/components/src/table/Table.module.css

```
@@ -98,9 +98,9 @@
 
   &.striped {
     .row {
-      &:nth-child(even),
+      &:nth-child(odd),
       &[data-even='false'] {
```

```suggestion
      &:nth-child(odd):not([data-even='false']),
      &[data-even='true'] {
```
hur ologiskt detta än ser ut så borde vi göra typ såhär